d3a_interface/sim_results/__init__.py

```diff
@@ -50,6 +50,17 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def is_house_node_type(area):
+    return get_unified_area_type(area) == "Area"
```

Comment:
Where is this method used? I can't seem to find it.

Reply:
Added and skipped its usage down the road while implementing. Thanks for catching will remove it now.